skbio/stats/composition.py

```diff
@@ -548,6 +551,12 @@ def ilr(mat, basis=None, check=True):
     >>> ilr(x)
     array([-0.7768362 , -0.68339802,  0.11704769])
 
+    Notes
+    -----
+    If the `basis` parameter is specified, it is expected to be a basis in the
+    Aitchison simplex.  If there are `D-1` elements specified in `mat`, then
+    the dimensions of the basis needs be `D-1 x D`, where rows represent
+    basis vectors, and the columns represent proportions.
     """
     mat = closure(mat)
     if basis is None:
```

Comment:
Commenting here because GitHub...

Just after line 563 we should perform the same check for 2D as in `ilr_inv`.